charts/merlin/templates/merlin-deployment.yaml

```diff
@@ -275,6 +275,8 @@ spec:
             value: {{ include "common.set-value" (list .Values.mlpApi.apiHost $globMlpApiHost) }}
           - name: MLFLOW_TRACKING_URL
             value: "{{ .Values.mlflow.trackingURL }}"
+          - name: ARTIFACT_SERVICE_TYPE
```

Comment:
For consistency, can we also append the `MLFLOW_` prefix to the variable name? The same change probably also needs to be done in the Merlin PR too.

Reply:
sure, no problem 😁